sdk/communication/communication-sms/samples/v1/typescript/src/usingAadAuth.ts

```diff
@@ -38,21 +38,29 @@ export async function main() {
   const credential: TokenCredential = isNode
     ? new DefaultAzureCredential()
     : new ClientSecretCredential(
-        process.env.AZURE_TENANT_ID,
-        process.env.AZURE_CLIENT_ID,
-        process.env.AZURE_CLIENT_SECRET
-      );
+      process.env.AZURE_TENANT_ID,
+      process.env.AZURE_CLIENT_ID,
+      process.env.AZURE_CLIENT_SECRET
+    );
 
   // create new client with endpoint and credentials
   const client = new SmsClient(endpoint, credential);
 
   // construct send request
+  var phoneNumbers: string[];
+  if (process.env.TO_PHONE_NUMBERS != undefined) {
+    phoneNumbers = process.env.TO_PHONE_NUMBERS.split(",")
+  }
+  else if (process.env.AZURE_PHONE_NUMBER != undefined) {
+    phoneNumbers = [process.env.AZURE_PHONE_NUMBER]
+  }
+  else {
+    phoneNumbers = ["<to-phone-number-1>", "<to-phone-number-2>"]
+  }
+
```

Comment:
My three comments apply to all these blocks.

Reply:
Thanks Will!